### ISSUE TITLE
New Modal Winder Vertical Centering Doesn't Work Sometimes

### ISSUE THREAD

**Issue by timayhelen** [elegantthemes/Divi-Beta#36]
Sometimes a modal window will not center vertically after Fikri's recent change. Once the browser window is resized, they center correctly. 

![modals](https://cloud.githubusercontent.com/assets/3333028/14136943/50a7ef16-f61a-11e5-99a5-b0a68040303c.gif)

**Comment by SlavaET** [comment]
Fixed. I found some more modals which were not aligned properly Cache Notice for example, Add New Layout from Divi library, Reset Roles and all the modals in ePanel except for the Portability modal.
So, there are 2 branches:
This one fixes alignment on the Divi Builder pages including Library and Roles pages - https://github.com/elegantthemes/submodule-builder/pull/636
And this one fixes modals alignment in the ePanel - https://github.com/elegantthemes/submodule-epanel/pull/28
They're independent of each other.
